README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+
+# hw01
+
+My name is **Bingxin** (Janssie) Zhu. In this course, I  want to 
+
+* Learn how to code using R
+* Conduct data anaglsis using R 
```

Comment:
I appreciate how concise and clear the formatting of this list is-- it's easier to read then if the same information was made into a sentence. Minor editing comment -- typo with analysis